Build & run Android display-test-app test is failing

### Clear title

Build & run Android display-test-app test is failing with "Timed out!"

### Expected vs. actual behavior

Expected: The Build & run Android display-test-app test should pass.
Actual: It is failing with a "Timed out!" error in the log.

### Reproduction steps

1. `cd test-apps/display-test-app`
2. `rushx android:all`


### Minimal reproducible example

_No response_

### Stack traces and logs

```
Android test app Built.
Fixing path setting in emulator's ini files...
Starting Android emulator...
Emulator started.
Will download iModel.
IMJS_ITWIN_ID: 480b00ee-ef2a-407a-8db8-43cc920044d4
IMJS_IMODEL_ID: 0930a578-125e-40d1-b2d4-73825629616d
Installing apk /Volumes/Builds/vsts_work_a/3/s/test-apps/display-test-app/android/imodeljs-test-app/app/build/outputs/apk/debug/app-debug.apk...
Performing Streamed Install
Success
APK installed.
Starting display-test-app...
Stopping: com.bentley.imodeljs_test_app
Starting: Intent { act=android.intent.action.MAIN cmp=com.bentley.imodeljs_test_app/.MainActivity }
display-test-app started.
Waiting for first render to finish...
Timed out!
Starting display-test-app...
Stopping: com.bentley.imodeljs_test_app
Starting: Intent { act=android.intent.action.MAIN cmp=com.bentley.imodeljs_test_app/.MainActivity }
display-test-app started.
Waiting for first render to finish...
Timed out!
Stopping emulator...
Emulator stopped.
Stopping the adb daemon
adb daemon not running.
Done

##[error]Bash exited with code '1'.
```

### Environment

iTwin.js Version: master
Runtime (e.g., Node.js 20, Electron 28): Node.js v22.21.1
Operating System: macOS



### Priority / urgency

If this is failing all the time, it is urgent.

### AI acknowledgment

- [ ] I used AI to generate this issue report.
- [ ] (If the above is checked) I have reviewed the AI-generated content before submitting.

--- a/common/config/azure-pipelines/templates/core-build.yaml
+++ b/common/config/azure-pipelines/templates/core-build.yaml
@@ -60,15 +60,16 @@ steps:
       RUSH_BUILD_CACHE_ENABLED: ${{parameters.rushBuildCacheEnabled}}
       VITE_CI: true
 
-  # - script: npm run android:all
-  #   workingDirectory: test-apps/display-test-app
-  #   displayName: Build & run Android display-test-app
-  #   env:
-  #     IMJS_OIDC_CLIENT_ID: $(IMJS_OIDC_CLIENT_ID)
-  #     IMJS_OIDC_CLIENT_SECRET: $(IMJS_OIDC_CLIENT_SECRET)
-  #     AZURE_DEVOPS_EXT_PAT: $(System.AccessToken)
-  #     TOKEN: $(GitHubPAT)
-  #   condition: and(succeeded(), ${{ parameters.buildMobile }}, eq(variables['Agent.OS'], 'Darwin'))
+  - script: npm run android:all
+    workingDirectory: test-apps/display-test-app
+    displayName: Build & run Android display-test-app
+    env:
+      IMJS_OIDC_CLIENT_ID: $(IMJS_OIDC_CLIENT_ID)
+      IMJS_OIDC_CLIENT_SECRET: $(IMJS_OIDC_CLIENT_SECRET)
+      AZURE_DEVOPS_EXT_PAT: $(System.AccessToken)
+      TOKEN: $(GitHubPAT)
+    # TODO: Re-enable once the Android crash after the OpenSSL update is resolved.
+    condition: and(succeeded(), false, ${{ parameters.buildMobile }}, eq(variables['Agent.OS'], 'Darwin'))
 
   - script: npm run ios:all
     workingDirectory: test-apps/display-test-app

--- a/common/config/azure-pipelines/templates/core-build.yaml
+++ b/common/config/azure-pipelines/templates/core-build.yaml
@@ -60,15 +60,15 @@ steps:
       RUSH_BUILD_CACHE_ENABLED: ${{parameters.rushBuildCacheEnabled}}
       VITE_CI: true
 
-  - script: npm run android:all
-    workingDirectory: test-apps/display-test-app
-    displayName: Build & run Android display-test-app
-    env:
-      IMJS_OIDC_CLIENT_ID: $(IMJS_OIDC_CLIENT_ID)
-      IMJS_OIDC_CLIENT_SECRET: $(IMJS_OIDC_CLIENT_SECRET)
-      AZURE_DEVOPS_EXT_PAT: $(System.AccessToken)
-      TOKEN: $(GitHubPAT)
-    condition: and(succeeded(), ${{ parameters.buildMobile }}, eq(variables['Agent.OS'], 'Darwin'))
+  # - script: npm run android:all
+  #   workingDirectory: test-apps/display-test-app
+  #   displayName: Build & run Android display-test-app
+  #   env:
+  #     IMJS_OIDC_CLIENT_ID: $(IMJS_OIDC_CLIENT_ID)
+  #     IMJS_OIDC_CLIENT_SECRET: $(IMJS_OIDC_CLIENT_SECRET)
+  #     AZURE_DEVOPS_EXT_PAT: $(System.AccessToken)
+  #     TOKEN: $(GitHubPAT)
+  #   condition: and(succeeded(), ${{ parameters.buildMobile }}, eq(variables['Agent.OS'], 'Darwin'))
 
   - script: npm run ios:all
     workingDirectory: test-apps/display-test-app

